engine/worker/builtin_gitclone.go

```diff
@@ -354,10 +357,14 @@ func extractInfo(w *currentWorker, dir string, params *[]sdk.Parameter, branch,
 		} else {
 			sendLog("git.branch: [empty]")
 		}
-	} else {
+	} else if branch != "" && branch != "{{.git.branch}}" {
 		sendLog(fmt.Sprintf("git.branch: %s", branch))
 	}
 
+	if tag != "" && tag != "{{.git.tag}}" {
```

Comment:
check tag != "{{.git.tag}}" to remove. .git.tag is not a default value on action GitClone